app/extraResources/walletSettings.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Do we need to check in this file?

Reply:
I has some hints about what configuration keys the application requires